src/frame/contracts.rs

```diff
@@ -116,6 +116,7 @@ pub struct InstantiatedEvent<T: Contracts> {
 }
 
 #[cfg(test)]
+#[cfg(feature = "integration-tests")]
```

Comment:
All tests in this module are integration tests so `cargo check --tests` will warn about unused items without this.

Reply:
Means the individual test ones are redundant now, but anyway I'll fix that in #165